website/components/live-example/LiveExample.tsx

```diff
@@ -141,6 +141,10 @@ function LiveExample<ComponentProps extends ComponentPropsInterface>({
   const [props, setProps] = useState<ComponentProps>(initialProps);
 
   const onChange = <T extends PropsType['default']>(value: T, prop: string) => {
+    if (value === '') {
```

Comment:
why disallow empty string from being a valid value for all knobs?

Reply:
This was supposed to remediate the bugs caused by the Select placeholder. I suppose I can move this logic to the Select knob itself?

Reply:
yeah a more focused change would be preferable

Reply:
done